### ISSUE TITLE
Fan Control tries to open outside of desktop (v162 and/or v163)

### ISSUE THREAD

**Issue by asmasca** [Rem0o/FanControl.Releases#1920]
Lately, Fan Control had been doing something weird. Each time I tried to open the window it starts outside of the desktop, making it innacccesible. I think it started when it updated to v162, but the behaviour continued to v163. 

I found that the issue was with the Height/Left coordinates of the CACHE file in the Configurations folder. It was telling the app to open at -1968, which is fully outside of the desktop. I manually tweaked them and now it shows fine. 

I'm using a 4K screen with 200% scale on Windows (maybe that's the issue with the weird coordinates). 

Could be worth tweaking the default coordinates.

**Comment by Rem0o** [comment]
Do you have a second monitor/virtual monitor or anything of that sort?

**Comment by asmasca** [comment]
Yes, I do have a second monitor (to the left of the main monitor). The app window wasn't rendering in that one either.

**Comment by Rem0o** [comment]
Minus values are relative to the main monitor, so a negative value means to the left of the main monitor, which fits your second monitor. I don't know exactly how that value got there, but it's the coordinate of the top left corner of the window.
